pycqed/measurement/pulse_sequences/multi_qubit_tek_seq_elts.py

```diff
@@ -1042,12 +1070,15 @@ def multi_parity_multi_round_seq(ancilla_qubit_names,
         end_pulses = [deepcopy(operation_dict[op]) for op in end_sequence]
 
         for pulse in end_pulses:
+            # if parity_loops > 0:
             pulse['ref_pulse'] = f'first_ro_{rounds}' + \
                                  f'_loop_{parity_loops-1}_tomo_{t}'
-            pulse['pulse_delay'] = 300e-9
-            pulse['ref_point'] = 'end'
+            pulse['pulse_delay'] = (16/2.4)*168e-9 - \
```

Comment:
would be even better if those numbers had names to that we know what they do :)

Reply:
I agree, want to add them? :))

Reply:
We now removed this function.